integration_tests/src/utils.rs

```diff
@@ -336,17 +336,19 @@ pub fn test_all_internet_connectivity(namespaces: NamespaceInfo) {
                 ],
             )
             .unwrap();
-            if String::from_utf8(out.stdout)
-                .unwrap()
-                .contains("1 received")
-            {
+            let output_string = from_utf8(&out.stdout).unwrap();
+            if output_string.contains("1 received") {
                 info!("Ping test passed for {}!", ns.get_name());
                 break;
             } else {
-                if Instant::now() - start > Duration::from_secs(60) {
+                if Instant::now() - start > Duration::from_secs(600) {
```

Comment:
10min wait intentionally left in? Usually the 1min was fine for github actions suite

Reply:
yup you're right here. Will fix